boards/arm/colibri_imx7d_m4/pinmux.c

```diff
@@ -11,6 +11,15 @@ static int colibri_imx7d_m4_pinmux_init(struct device *dev)
 {
 	ARG_UNUSED(dev);
 
+#ifdef CONFIG_GPIO_IMX_PORT_2
+	/* GPIO2_IO26 Mux Config */
+	*(&IOMUXC_SW_MUX_CTL_PAD_EPDC_GDRL) = 5;
+	*(&IOMUXC_SW_PAD_CTL_PAD_EPDC_GDRL) =
+		IOMUXC_SW_PAD_CTL_PAD_EPDC_GDRL_PS(2) |
+		IOMUXC_SW_PAD_CTL_PAD_EPDC_GDRL_PE_MASK |
+		IOMUXC_SW_PAD_CTL_PAD_EPDC_GDRL_HYS_MASK;
```

Comment:
Same question as before about getting a reference and immediately dereferencing it. Why not use `IOMUXC_SW_MUX_CTL_PAD_EPDC_GDRL` and `IOMUXC_SW_PAD_CTL_PAD_EPDC_GDR` directly?